docs/reference/instrumentation/metrics.md

```diff
@@ -39,3 +39,5 @@ You can query the metrics endpoint for these components using an HTTP scrape, an
 | federatedhpa_pull_metrics_duration_seconds      | Histogram         | Duration in seconds taken by the FederatedHPA to pull metrics. By the result, 'error' means the FederatedHPA failed to pull the metrics. Otherwise 'success'.                       | result<br/>metricType                 | karmada-controller-manager                   |
 | pool_get_operation_total                        | Counter           | Total times of getting from pool                                                                                                                                               | name<br/>from                         | karmada-controller-manager<br/>karmada-agent |
 | pool_put_operation_total                        | Counter           | Total times of putting from pool                                                                                                                                               | name<br/>to                           | karmada-controller-manager<br/>karmada-agent |
+| recreate_resource_to_cluster                    | Counter           | Number of recreating operation of the resource to a target member cluster. By the result, 'error' means a resource recreated failed. Otherwise 'success'. Cluster means the target member cluster. | result<br/>apiversion<br/>kind<br/>cluster | karmada-controller-manager                   |
+| update_resource_to_cluster                      | Counter           | Number of updating operation of the resource to a target member cluster. By the result, 'error' means a resource updated failed. Otherwise 'success'. Cluster means the target member cluster.     | result<br/>apiversion<br/>kind<br/>cluster | karmada-controller-manager                   |
```

Comment:
```suggestion
| update_resource_to_cluster                      | Counter           | Number of resource updates to a target member cluster. By the result, 'error' means a resource updated failed. Otherwise 'success'. Cluster means the target member cluster.     | result<br/>apiversion<br/>kind<br/>cluster | karmada-controller-manager                   |
```

Reply:
thanks, done